jekyll.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency("addressable",           "~> 2.4")
   s.add_runtime_dependency("colorator",             "~> 1.0")
   s.add_runtime_dependency("em-websocket",          "~> 0.5")
-  s.add_runtime_dependency("i18n",                  "~> 1.0")
+  s.add_runtime_dependency("i18n",                  ">= 0.9.5")
```

Comment:
Do we want to include a pessimistic upper bound? `< 2.0.0`?

Reply:
Good idea! I will add that.